### DISCUSSION TITLE
java question

### DISCUSSION THREAD

**Discussion by sanula12** [sanula12/java_calculator#4] [Q&A] [answered]
what is awt in java

**Comment by mvinduwara** [comment, marked as answer]
Abstract Window Toolkit is referred to as AWT. Presented with the initial JDK (Java Development Kit) release, it is among the fundamental GUI (Graphical User Interface) toolkits in Java. Java programmes may design and handle graphical user interface elements like windows, buttons, text fields, and menus with the help of AWT, which is a collection of classes and methods. Because they depend on the native GUI libraries of each platform, these components are platform-dependent, which means that their look and behaviour may differ across different operating systems. Event listeners attached to GUI components can record and process events generated by user actions like mouse clicks and keyboard inputs thanks to AWT's event-handling features. Even while AWT was revolutionary at the time, it has mainly
